src/test/java/eu/xenit/gradle/alfrescosdk/AmpBasePluginTest.java

```diff
@@ -139,13 +133,10 @@ public void manualConfigureSourceSet() {
                 moduleProperties.setProperty("module.version", "1.0.0");
             });
 
-            ampConfig.web(sourceDirs -> {
```

Comment:
Action methods where removed because this changed.

Reply:
Reverted to original test